py/PulumiPlugin.yaml

```diff
@@ -1 +1,4 @@
-runtime: python
+runtime:
+  name: python
+  options:
+    virtualenv: venv
```

Comment:
Does this actually change anything? We looked at the code and we don't expect this would create venv for local packages....

Reply:
One +would+ need to activate the venv in order to execute the provider code, e.g. to call `GetSchema`. Therefore, `pulumi package add` should activate the venv. It should be activated for any execution of the provider code, right?
The debugging experience certainly needs the venv, for `debugpy` package.

Reply:
Alright, I confirmed that Pulumi does not activate the venv as one would expect.  I had done some manual activations and I guess I tricked myself.

But this seems like an important thing to fix, right?

Reply:
Yeah, this is not great. https://github.com/pulumi/pulumi/issues/18208 is kind of similar issue for Node, but maybe you could open a new one for Python and articulate that this is important for `package add`/`pulumi install` use cases.

Reply:
Filed: https://github.com/pulumi/pulumi/issues/19233